### DISCUSSION TITLE
winrt appears to not be on the python Package Index

### DISCUSSION THREAD

**Discussion by a-usr** [pywinrt/pywinrt#38] [Q&A] [answered]
As the title says, it appears that winrt is not on the index. pip cant find winrt, and on the PyPI website i can only find a version from 2 years ago. Do i have to configure pip in some special way to be able to download winrt?

**Comment by dlech** [comment, marked as answer]
The package has been split into individual packages for each WinRT namespace, e.g. https://pypi.org/project/winrt-Windows.Foundation/
